test/sycl_array_zero_init.ll

```diff
@@ -0,0 +1,56 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+
```

Comment:
Please add a check to make sure that after round-trip translation we still have the same llvm.memcpy intrinsic, not llvm.memset